pyppeteer/target.py

```diff
@@ -17,33 +19,46 @@
 class Target(object):
```

Comment:
Inheriting from object isn't necessary since Python 3
```suggestion
class Target:
```